sources/packages/backend/apps/load-test-gateway/src/utils/auth.constants.ts

```diff
@@ -0,0 +1 @@
+export const IS_PUBLIC = "isPublic";
```

Comment:
I do not anticipate a public endpoint for this API. I believe that it can be added if needed.